python/dgl/core.py

```diff
@@ -213,7 +213,8 @@ def data_dict_to_list(graph, data_dict, func, target):
         The input graph.
     data_dict : dict[str, Tensor] or dict[(str, str, str), Tensor]]
         Node or edge data stored in DGLGraph. The key of the dictionary
-        is the node type name or edge type name.
+        is the node type name or edge type name. If there is only 1 source
+        node, data_dict is the value of feature not a dict.
```

Comment:
`is the value of feature` indicates `Tensor`? If yes, explain more clearly in docstring?

Reply:
Modified